CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
-### 2.0.1 (Next)
+### 2.1.1 (Next)
 
+* [#436](https://github.com/slack-ruby/slack-ruby-client/pull/436): Remove dependency on unused websocket-driver - [@blowfishpro](https://github.com/blowfishpro).
 * Your contribution here.
+
+### 2.1.0 (2022-11-29)
```

Comment:
We haven't shipped 2.1.0 yet: move your CHANGELOG line under 2.1.0 (Next).

Reply:
Whoops, I guess I misunderstood the last PR and didn't actually check releases.  I'll fix that.

Reply:
Fixed